TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt

```diff
@@ -88,28 +90,28 @@ class ParserTest {
     }
 
     @Test fun quoteBulletPointsCode() {
-        val text = "Complex:\n> Quote\nWith points:\n+ bullet `one`\n* bullet `two` is `long`"
+        val text =  """Complex:$separator> Quote${separator}With points:$separator+ bullet `one`$separator* bullet `two` is `long`"""
```

Comment:
let's decrease the number of quotes here

Reply:
Done